pkg/webhook/server/server.go

```diff
@@ -94,6 +97,9 @@ type Server struct {
 	// Log is an optional logger to write informational and error messages to.
 	// If not specified, no messages will be logged.
 	Log logr.Logger
+
+	// CipherSuites is a comma separated list of TLS Cipher Suite names
+	CipherSuites string
```

Comment:
Can you make this a `[]string` and move the string splitting into the `cmd/` package? It makes it easier for others to re-use this package in future 😄

Reply:
Also, what's the default behaviour if this struct leaves the field as `nil`/empty? The struct itself should have some sensible default behaviour too (probably just leaving the field unset and deferring to Go's standard behaviour?)

Reply:
hmm, yeah, I'd accounted for the flag not being passed but not CipherSuites not being set at all presumably by a different caller?

I guess I don't want people to be confused by setting tlsCipherSuites to empty (perhaps if TLS1.2 ciphers are all vulnerable and only TLS1.3 works now) and getting a list of ciphers they didn't want. 

So I'm not really sure how to distinguish tls-cipher-suites being unset (and getting the default list from main.go) vs tls-cipher-suites being explicitly set to empty in server.go.

Reply:
You should be able to check for explicit 'empty' vs 'unset' by checking explicitly for 'nil': https://play.golang.org/p/niN-dHOYyLF

Then if you could add a comment to the field explaining "To disable all TLS 1.2 cipher suites, specify the empty slice (`[]string{}`)."

Reply:
ok, I just tested this - thinking about it, it should have been more obvious to me - go is terrible at empty values, so it sees an empty value ([]uint16{}) for tls.Config.CipherSuites and goes with the default list of cipher suites - so setting an empty value is less secure than one strong cipher.